### ISSUE TITLE
dr_wav append write?

### ISSUE THREAD

**Issue by Youlean** [mackron/dr_libs#211]
Hi, is there a way for dr_wav to append data to a wav file when writing instead of rewriting the file?

**Comment by mackron** [comment]
No, unfortunately appending is not supported. The writing API was only really designed for basic use cases.

**Comment by mackron** [comment]
Yeah I'm not against adding support or anything. I'll mark this as a feature request and get to it when I can, but it could be a ways away.

**Comment by Youlean** [comment]
I qm already working on it. Might have it ready in a couple of hours

**Comment by Youlean** [comment]
I have run some tests today, and it seems it's working as expected.
